src/camera_path.cu

```diff
@@ -99,6 +102,8 @@ void from_json(bool is_first, const json& j, CameraKeyframe& p, const CameraKeyf
 	j.at("scale").get_to(p.scale);
 	j.at("fov").get_to(p.fov);
 	if (j.contains("dof")) j.at("dof").get_to(p.aperture_size); else j.at("aperture_size").get_to(p.aperture_size);
+	if (j.contains("glow_mode")) j.at("glow_mode").get_to(p.glow_mode); else return;
```

Comment:
Could you replace these return statements with default values for glow_mode and glow_y_cutoff (both zero)? Then the PR will be good to merge. Thanks!

Reply:
If I understand how `get_to` is working, this should be set now!